src/components/SummaryBar/SummaryBar.jsx

```diff
@@ -289,8 +290,8 @@ const SummaryBar = props => {
                 </font>
                 <CardTitle className={`align-middle ${darkMode ? 'text-light' : 'text-dark'}`} tag="h3">
                   <div className='font-weight-bold'>
-                    {displayUserProfile.firstName + ' '}
-                    {displayUserProfile.lastName}
+                    {userProfile?.firstName ||displayUserProfile.firstName + ' '}
+                    {userProfile?.lastName || displayUserProfile.lastName}
```

Comment:
nit: space between Curly braces and the variable. ESLint gives me a warning if there's no space. Same for line 293

Reply:
The space helps to separate first name and lastname

Reply:
I meant add a space between "{" and "userProfile?.firstName".